docs/docsite/rst/playbooks_variables.rst

```diff
@@ -1006,21 +1002,18 @@ This makes it clear to the playbook reader that you've made a conscious choice t
 configuration that the role can't assume by itself.  It also allows you to pass something site-specific that isn't really part of the
 role you are sharing with others.
 
-This can often be used for things that might apply to some hosts multiple times,
-like so::
+This can often be used for things that might apply to some hosts multiple times. For example::
 
     roles:
        - { role: app_user, name: Ian    }
        - { role: app_user, name: Terry  }
        - { role: app_user, name: Graham }
        - { role: app_user, name: John   }
 
-That's a bit arbitrary, but you can see how the same role was invoked multiple times.  In that example it's quite likely there was
+In this example, the same role was invoked multiple times.  In that example it's quite likely there was
```

Comment:
Should the second sentence be changed as well.  perhaps:
```
In this example, the same role was invoked multiple times.  Judging from the parameters the role is being passed, it's quite likely there was[...]
```

Reply:
Got it. Thanks